src/magic.h

```diff
@@ -227,6 +227,8 @@ class spell_type
 
         // base amount of time to cast the spell in moves
         int base_casting_time;
+        // If spell is to summon a vehicle, the vproto_id of the vehicle
+        std::string vehicle_id;
```

Comment:
If it's supposed to be a `vproto_id`, why is it a `std::string` and not a `vproto_id`?

Reply:
resolved as per reusing effect_str as Korggent suggests